README.md

```diff
@@ -122,57 +191,154 @@ MissionUpload test result: Failed
+Checking if contrast is a possible setting.
+Getting option for contrast
+Getting range properties for contrast
+Setting contrast to 0
+Getting option for contrast
+Setting contrast to 255
+[09:41:33|Warn ] sending again, retries to do: 2  (528). (mavlink_commands.cpp:222)
+Getting option for contrast
+Setting contrast to 127
+Getting option for contrast
+Resetting setting to initial option.
+Setting contrast to 32
+CameraSettings test result: Success
+----
+[09:41:34|Debug] request camera info (camera_impl.cpp:142)
+[09:41:34|Warn ] sending again, retries to do: 1  (528). (mavlink_commands.cpp:222)
+[09:41:34|Debug] request camera info (camera_impl.cpp:142)
+[09:41:34|Error] Retrying failed (528) (mavlink_commands.cpp:239)
+[09:41:34|Debug] downloading camera definition file from: http://127.0.0.1:8000/camera-def-uvc.xml (camera_impl.cpp:931)
+[09:41:34|Info ] Downloading camera definition from: http://127.0.0.1:8000/camera-def-uvc.xml (camera_impl.cpp:1171)
+[09:41:35|Warn ] sending again, retries to do: 3  (528). (mavlink_commands.cpp:222)
+[09:41:36|Warn ] sending again, retries to do: 2  (528). (mavlink_commands.cpp:222)
+[09:41:36|Warn ] sending again, retries to do: 1  (528). (mavlink_commands.cpp:222)
+[09:41:37|Error] Retrying failed (528) (mavlink_commands.cpp:239)
+[09:41:37|Debug] downloading camera definition file from: http://127.0.0.1:8000/camera-def-uvc.xml (camera_impl.cpp:931)
+[09:41:37|Info ] Downloading camera definition from: http://127.0.0.1:8000/camera-def-uvc.xml (camera_impl.cpp:1171)
+Checking if brightness is a possible setting.
+Getting option for brightness
+Getting range properties for brightness
+Setting brightness to 0
+Getting option for brightness
+Setting brightness to 225
+Getting option for brightness
+Setting brightness to 112
+Getting option for brightness
+Resetting setting to initial option.
+Setting brightness to 128
 CameraSettings test result: Success
 ----
 Test summary:
 CameraSettings:
 name: CameraSettings
 camera_id: 2
 param_name: wb-mode
-param_value: 0
-sub_param_name: wb-temp
-sub_param_values: 4000
-sub_param_value:
+param_is_range: false
+  => passed
+----
+CameraSettings:
+name: CameraSettings
+camera_id: 2
+param_name: exp-mode
+param_is_range: false
+  => passed
+----
+CameraSettings:
+name: CameraSettings
+camera_id: 2
+param_name: contrast
+param_is_range: true
+  => passed
+----
+CameraSettings:
+name: CameraSettings
+camera_id: 2
+param_name: brightness
+param_is_range: true
   => passed
 ----
 ```
+
+**Note:**
+
+In the test above some errors are logged but can be ignored because there is no autopilot connected:
+```
+[09:41:35|Warn ] sending again, retries to do: 3  (528). (mavlink_commands.cpp:222)
+[09:41:36|Warn ] sending again, retries to do: 2  (528). (mavlink_commands.cpp:222)
+[09:41:36|Warn ] sending again, retries to do: 1  (528). (mavlink_commands.cpp:222)
+[09:41:37|Error] Retrying failed (528) (mavlink_commands.cpp:239)
```

Comment:
Will this be fixed in the SDK?

Reply:
At some point. We need to figure out how to properly deal with systems that don't have an autopilot, or at least we just don't show warnings for this sort of error.